frontend/packages/ux-editor/src/components/config/FormComponentConfig.tsx

```diff
@@ -141,6 +142,7 @@ export const FormComponentConfig = ({
           propertyKey={propertyKey}
           defaultValue={properties[propertyKey].default}
           key={propertyKey}
+          helpText={getHelpText(propertyKey)}
```

Comment:
This `helpText` prop is not necessary, the `useComponentPropertyHelpText` is called in `EditBooleanValue` component with `propertyKey` already 🙂

Reply:
I removed it 👍